### PR TITLE
Fix bug Where timestamps! causes other Dolly::Document classes to have timestamps completes #108

### PR DESCRIPTION
**Pull request by seancookr**

### CODE REVIEW

File: test/document_test.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def add_to_foo key, value
 end
 
 class WithTime < Dolly::Document
-  property :created_at
+  property :created_at, default: -> {DateTime.now}
```
Comment (seancookr) [inline test/document_test.rb]: This fixes a test that should have been failing the entire time, but was passing until I made my changes.

Comment (netlogics) [inline test/document_test.rb]: 👏

**Comment by netlogics** [comment]
:shipit:

File: lib/dolly/document.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ class Document
     attr_accessor :rows, :doc, :key
     class_attribute :properties
     cattr_accessor :timestamps
+    @@timestamps = {}
```
Comment (neojin) [inline lib/dolly/document.rb]: from http://apidock.com/rails/Class/cattr_accessor

```
Also you can pass a block to set up the attribute with a default value.

class Person
  cattr_accessor :hair_colors do
    [:brown, :black, :blonde, :red]
  end
end

Person.class_variable_get("@@hair_colors") #=> [:brown, :black, :blonde, :red]
```

File: test/document_test.rb (hunk 1 above)
Comment (neojin) [inline test/document_test.rb]: why not `Time` ?
http://stackoverflow.com/questions/1261329/whats-the-difference-between-datetime-and-time-in-ruby/21075654#21075654

Comment (seancookr) [inline test/document_test.rb]: @neojin I did not use `Time` as when i first added the feature nobody commented on my use of `DateTime`. I can switch it.

**Comment by seancookr** [comment]
maybe I should merge this to a `1.1` branch?

**Comment by netlogics** [comment]
Merging to 1.1 seems safest. Another option would be to separate concerns and use versioning.
- revert the last commit `f015b1d`
- update the version number to 1.0.1
- merge this PR to 1.0
- tag v1.0.1
- submit a new PR with the reverted commit against the 1.0 branch
- update the version number to 1.0.2
- merge
- tag as v1.1.0 or v1.0.2 depending.

Consequentially this would require consumers that only specify a branch name in their Gemfiles to use tags and or specify the version number.

**Comment by seancookr** [comment]
@javierg @neojin ☝️ what do you think regarding @netlogics suggestions? Trying to determine the proper course of action here so as not to break things

**Comment by neojin** [comment]
I'll defer to you guys as I don't know enough about your release process

**Comment by javierg** [comment]
I would  merge it I'm 1.1 and start using semantic versioning after this

**Comment by seancookr** [comment]
Reopening this against `1.1`

**Pull request closed without merging**